wordle-web/pages/ScoreStats.vue

```diff
@@ -16,12 +16,14 @@
         <v-simple-table>
           <thead>
             <tr>
+              <th>Name</th>
               <th>Score</th>
               <th>Avg. Seconds</th>
             </tr>
           </thead>
           <tbody>
             <tr v-for="(stat, index) in stats" :key="index">
+              <td>{{ stat.name }}</td>
```

Comment:
There is no name property on the `ScoreStat` object being returned in your TS below. You should be returning `Player` objects here.